examples/python/dataframe_query/README.md

```diff
@@ -0,0 +1,16 @@
+This example will query for the first 10 rows of data in your recording of choice,
+and display the results as a table in your terminal.
+
+You can use one of your recordings, or grab one from our hosted examples, e.g.:
+```bash
+curl 'https://app.rerun.io/version/latest/examples/dna.rrd' -o - > /tmp/dna.rrd
+```
+
+The results can be filtered further by specifying an entity filter expression:
+```bash
+python dataframe_query.py my_recording.rrd /skeleton/left_shoulder/**\
```

Comment:
nit: how about you continue with the example above (so it's easier for someone to follow a working example with downloaded recording), so here you have:
```
python dataframe_query.py /tmp/dna.rrd /helix/structure
```